dango/testing/tests/httpd.rs

```diff
@@ -0,0 +1,131 @@
+use {
+    actix_web::{
+        body::MessageBody,
+        dev::{ServiceFactory, ServiceRequest, ServiceResponse},
+        App,
+    },
+    assertor::*,
+    dango_httpd::{
+        graphql::{build_schema, types::transfer::Transfer},
+        server::config_app,
+    },
+    dango_testing::setup_test_with_indexer,
+    dango_types::{
+        account::single,
+        account_factory::{self, AccountParams},
+    },
+    grug::{
+        build_actix_app_with_config, call_graphql, setup_tracing_subscriber, Coins,
+        GraphQLCustomRequest, Message, NonEmpty, PaginatedResponse, ResultExt,
+    },
+    indexer_httpd::context::Context,
+};
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+async fn graphql_returns_transfer() -> anyhow::Result<()> {
+    setup_tracing_subscriber(tracing::Level::INFO);
+
+    let ((mut suite, mut accounts, _, contracts), indexer_context) = setup_test_with_indexer();
+
+    // Copied from benchmarks.rs
+    let msgs = vec![Message::execute(
+        contracts.account_factory,
+        &account_factory::ExecuteMsg::RegisterAccount {
+            params: AccountParams::Spot(single::Params::new(accounts.user1.username.clone())),
+        },
+        Coins::one("hyp/eth/usdc", 100_000_000).unwrap(),
```

Comment:
Instead of magic string `"hyp/eth/usdc"`, use `dango_testing::constants::USDC_DENOM` so the test doesn't break in the future. (We will be changing USDC denom to the alloyed denom `"alloy/usdc"`)